dls-controls/malcolmjs
# Feature/method post #169

## Description

Methods now send posts, handle corresponding returns and display results.
Various tweaks with method input widget functionality and some other bugfixes and error handling.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- view widgets for "save" method on localhost:3000/PANDA

testing of method return behaviour would require connecting to a pymalcolm server; this can be done by running DEMO-HELLO.yaml as per the pymalcolm tutorial docs [here](http://pymalcolm.readthedocs.io/en/latest/tutorials/hello.html), setting malcolmjs to connect to port 8008 for ws communications and then navigating to localhost:3000/HELLO 

## Agile board tracking

connect to #169 
connect to #170 
connect to #171
